src/OhioBox.Metrics/Reporters/Graphite/GraphiteMetricsReporterFactory.cs

```diff
@@ -13,6 +13,12 @@ public GraphiteMetricsReporterFactory(GraphiteConfiguration configuration, IGrap
 			_graphiteSwitch = graphiteSwitch;
 		}
 
+		public GraphiteMetricsReporterFactory(string ipAddress, string prefix = "" , bool reportMachineName = false, IGraphiteSwitch graphiteSwitch = null)
```

Comment:
Why not add default values to GraphiteConfiguration object?
We are using an object to keep the API backword compatible

Reply:
the UX and discoverability of doing it that way is way higher than building an entire object just to discover what config parameters i need, especially since most of the times for simple cases we just need to supply ipAddress.

i accept that the GraphiteConfiguration is a good solution when needed. but it doesn't mean it has to be the only solution. 

WDYT?

Reply:
I think that using a constructor will lead to a more complex configuration at the long run, since it breaks the encapsulation of the configuratoin.
Whats so painful in creating an object (the default values can be initiated in his ctor the same why), and it would keep backward compatibility, which is the main issue here

Reply:
the pain is in lack of simplicity and discoverability. 

lack of simplicity - it just more code and more mental leaps to create a new graphite reporter.
discoverability - because from the config class i cannot understand with ease what is a must and what is not. 

regarding, "breaks the encapsulation of the configuration", i don't understand what it means? can you describe  a sensible scenario in which something will "break".